lib/linter-plus.coffee

```diff
@@ -110,6 +112,6 @@ class Linter
     @views.destroy()
     @linters.deactivate()
     @commands.destroy()
-    @messages.destroy()
+    @messages.deactivate()
```

Comment:
we should really rename all these things dispose so that we are implementing the disposable interface.